plugins/build-your-own.rst

```diff
@@ -4,7 +4,7 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
```

Comment:
this will only be correct once we have released 1.1 right? we might want to add a `versionadded` block in strategically relevant places to explain what was changed in 1.1